### DISCUSSION TITLE
网站评论时显示duplicate key value violates unique constraint "wl_comment_pkey"

### DISCUSSION THREAD

**Discussion by dyxang** [walinejs/waline#1135] [Q&A] [answered]
相近问题#1100 已解决。但评论区无法创建新评论。
部署环境：
- `@waline/vercel@1.18.7`
- 客户端引入`https://unpkg.com/@waline/client@2.6.1/dist/waline.js`

问题复现：

1. 从使用Leancloud的最新服务端导出
2. 新建部署，然后注册进入后台。（注册时填的信息和原来服务端注册信息一样）
3. 导入至Postgresql，后台显示成功
4. 在网站评论区进行评论（例子：[链接](https://www.1inn.top/post/%E4%B8%89%E4%B8%AA%E6%9C%88%E7%9A%84%E6%80%BB%E7%BB%93/)），显示`duplicate key value violates unique constraint "wl_comment_pkey"`
5. 控制台报错:
```
POST https://p1nlun.1inn.top/comment?lang=zh-CN 500
(匿名) @ fetch.ts:144
T @ fetch.ts:148
e.config.login.e.isLogin.Lo.onClick.t.<computed>.t.<computed> @ emoji.ts:42
Ht @ index.ts:29
Wt @ index.ts:29
n @ index.ts:29
```

**Comment by lizheming** [comment, marked as answer]
https://blog.csdn.net/weixin_43064185/article/details/109181428 可以先按照网上的方法自行尝试一下试试

**Comment by dyxang** [comment]
昨天太晚没去试原先问题的解决方法抱歉。**提醒小白，教程里不能生搬硬套。**
文章里解决方法的命令行是：
`SELECT setval('表名_id_seq', (SELECT MAX(id) FROM 表名)+1)`
但是在**Supbase**上应该是：
`SELECT setval('wl_comment_seq', (SELECT MAX(id) FROM wl_comment)+1)`
我也不懂，可能是个例也可能是普遍现象
